src/routes/terms/+page.markdoc

```diff
@@ -64,50 +54,67 @@ You agree not to disclose your password to any third party. You must notify us i
 
-Notwithstanding any other provision of these Terms, to the maximum extent permitted by law, we may collect, generate, process, use and/or publish Anonymous Information (defined below) relating to your, your users’ use of the Service, and/or disclose it, in order to provide, improve and publicize the Service and our products and services, in order to track general industry trends, develop and publish white papers, reports and summaries, and for other legitimate business purposes, all the foregoing without restriction or limitation of any kind.  “Anonymous Information” means information that does not enable the identification of an individual, such as aggregated and analytics information. For clarity, Anonymous Information will not include any personally identifiable information, and Appwrite shall own all Anonymous Information collected or obtained by Appwrite.
+Notwithstanding any other provision of these Terms, to the maximum extent permitted by law, we may collect, generate, process, use, and/or publish Anonymous Information (defined below) relating to your and your users’ use of the Service, and/or disclose it, in order to provide, improve, and publicize the Service and our products and services, track general industry trends, develop and publish white papers, reports, and summaries, and for other legitimate business purposes, all without restriction or limitation of any kind. "Anonymous Information" means information that does not enable the identification of an individual, such as aggregated and analytics information. For clarity, Anonymous Information will not include any personally identifiable information, and Appwrite shall own all Anonymous Information collected or obtained by Appwrite.
 
-# Prohibited Use of the SMS Messages Service
+# Prohibited Use
 
-You agree not to use the Service for the sending of unsolicited or spam SMS messages. This includes but is not limited to the prohibition of 
-- using the Service in connection with any content, product, or service the recipient has not explicitly requested.
-- sending SMS messages through the Service unless you have obtained prior express consent from the recipients, as required by applicable law.
+You agree not to use the Service for any unlawful or prohibited activities. This includes, but is not limited to:
 
-Failure to adhere to these prohibitions and/or any applicable law relating to the use of such SMS Service can result in immediate termination of your account and further legal action. You shall comply with any and all applicable laws and regulations. 
+- **Illegal Content**: Hosting, transmitting, or distributing content that is illegal, infringing, harmful, threatening, defamatory, obscene, harassing, or otherwise objectionable.
```

Comment:
Let's use the partial for this to reduce duplicate content. For example, see

https://github.com/appwrite/website/blob/b3ba3f944275aa4849b5c37266f33ceee32bf22a/src/routes/docs/advanced/platform/fair-use-policy/%2Bpage.markdoc#L23